### PR TITLE
Convert done callbacks to async/await in the last two unit test files

### PR DESCRIPTION
**Pull request by timvandermeij**
It's highly recommended to use the `?w=1` flag during review because of many indentation changes.

### CODE REVIEW

File: test/unit/evaluator_spec.js [hunk 1]
```diff
@@ -377,54 +341,58 @@ describe("evaluator", function () {
...
+        // Shouldn't get here.
+        expect(false).toEqual(true);
+      } catch (_) {
+        expect(!!result.fnArray && !!result.argsArray).toEqual(true);
+        expect(result.fnArray.length).toEqual(0);
+      }
     });
-    it("should abort text parsing parsing", function (done) {
+
+    it("should abort text parsing parsing", async function () {
```
Comment (Snuffleupagus) [inline test/unit/evaluator_spec.js:378]: Nit: Pre-existing issue, but there's probably one "parsing" too much in the test name :-)
I'm guessing that it *should* actually be:
```suggestion
    it("should abort text content parsing", async function () {
```

**Review by Snuffleupagus — APPROVED** [review]
Well, that certainly was *challenging* to try and review; I can only image that actually making the changes was even worse :-)

Looks great to me, and using `async`/`await` really is a much nicer format; thank you for modernizing the unit-tests!

**Comment by timvandermeij** [comment]
Thank you for the reviews!

**Comment by Snuffleupagus** [comment]
@timvandermeij Do you want to fix the font-tests as well, or should I do that?

**Comment by timvandermeij** [comment]
If you've already started on it, feel free to finish it. I won't have too much time the coming days, so otherwise it'll have to wait a couple of days.

**Pull request merged**